static/js/movie.js

```diff
@@ -1,35 +1,70 @@
 const reviewContainer = document.querySelector(".review-container");
 const screeningUI = document.querySelector('.screenings');
 const path = window.location.pathname;
+const movieId = path.split('/').pop();
 
-(async () => {
-    const id = path.split('/').pop();
-    const res = await fetch(`/api/movies/${id}/screenings`);
-    const data = await res.json();
+let currentPage = 1;
+let pageCount;
```

Comment:
Maybe have variable name that show that it is for the screenings, since the reviews also have pagination. To make the code more clear and easier to read.